### DISCUSSION TITLE
Gabarit par défaut - Logo Québec chevauche le titre du formulaire

### DISCUSSION THREAD

**Discussion by MJMESS** [MTESSDev/FRW#45] [Annonces]
Bonjour, 

est-ce prévu de faire une petite modification au gabarit par défaut afin que le logo du Québec ne chevauche pas le titre du formulaire dans le pdf généré ?

![image](https://user-images.githubusercontent.com/122536311/229602565-5996f5a5-a588-45e4-876d-4ff810de5a91.png)

Merci!

**Comment by j0ber** [comment]
Comme il semble que ce soit un bug, s.v.p. créer un issue.
![image](https://user-images.githubusercontent.com/26974817/229603683-4c40ff82-be58-46a9-9832-30d1e707a671.png)

**Comment by j0ber** [comment]
Transformé en issue. Voir #48.
